Essentials/src/messages.properties

```diff
@@ -525,14 +506,14 @@ teleportInvalidLocation=Value of coordinates cannot be over 30000000
 teleportNewPlayerError=\u00a74Failed to teleport new player\!
 teleportRequest=\u00a7c{0}\u00a76 has requested to teleport to you.
 teleportRequestAllCancelled=\u00a76All outstanding teleport requests cancelled.
-teleportRequestSpecificCancelled=\u00a76Outstanding teleport request with {0} cancelled.
+teleportRequestSpecificCancelled=\u00a76Outstanding teleport request with\u00a7c {0}\u00a76 cancelled.
 teleportRequestTimeoutInfo=\u00a76This request will timeout after\u00a7c {0} seconds\u00a76.
 teleportTop=\u00a76Teleporting to top.
 teleportToPlayer=\u00a76Teleporting to \u00a7c{0}\u00a76.
 tempbanExempt=\u00a74You may not tempban that player.
 tempbanExemptOffline=\u00a74You may not tempban offline players.
 tempbanJoin=You are banned from this server for {0}. Reason: {1}
-tempBanned=\u00a7cYou have been temporarily banned for {0}\:\n\u00a7r{2}
+tempBanned=\u00a7cYou have been temporarily banned for\u00a7r {0}\:\n\u00a7r{2}
```

Comment:
Do we really want to reset this? Seems like it would take color our of everything. Maybe if anything we want to re-apply styling to the : afterwards

Reply:
Yes, formatting to display the specified time format with a different color.

Reply:
Ok, looked back and 0 is the expiry time.